fetch.js

```diff
@@ -40,14 +40,28 @@ function add_table_row(handle, json){
 async function do_magic(){
     ipcRenderer.on('get-it', (event,message) =>
     {
-        const array = message.split(',')
-        console.log(array, "here")
-        const handle1 = array[0]
-        const repo1 = array[1]
-        handle = handle1
-        repo = repo1
-        console.log(handle,repo)
-        fetch_data_for_single_user(handle)    
+        console.log("hello")
+        //const array = message.split(',')
+        //console.log(array, "here")
+        // const handle1 = array[0]
+        // const repo1 = array[1]
+        // console.log(repo1)
+        // handle = handle1
+        // repo = repo1
+
+        console.log(message)
+        group=window.localStorage.getItem(message);
+        // console.log("helllo")
+        group2=JSON.parse(group)
+        //console.log(group2)
+
+         for (var i = 0; i < group2.length; i++){
+      var handle=group2[i].gh_handle;
+     var repo=group2[i].gh_handle+"/"+group2[i].repository;
```

Comment:
No need to do this. The user will give the repo in the format <org-name/username>/<repo name> because many people directly contribute to a particular organization as a member or collaborator.